src/Refinements/CstrHead.class.st

```diff
@@ -111,7 +111,7 @@ CstrHead >> sol1: k [
 	ys := pred args.
 	k1 = k ifFalse: [ ^#() ].
 	xs := k kArgs: ys.
-	^{ #() -> (xs collectWithIndex: [ :x :j | PAtomEq x: x toEVar y: (ys at: j) toEVar ]). }
+	^{ #() -> (xs collectWithIndex: [ :x :j | PAtom brel: #=== x: x toEVar y: (ys at: j) toEVar ]). }
```

Comment:
This PR introduces `Brel` and `Bop` pools, so why not do here:
```
...[ :x :j | PAtom brel: Eq x: x toEVar...
```

Reply:
I am not sure whether `Eq` or `#===` is primary here.  But it does not matter, because this is throw-away code anyway — pending investigation of #75 , #59  and friends.